worldedit-core/src/main/java/com/sk89q/worldedit/extension/platform/AbstractPlayerActor.java

```diff
@@ -482,6 +482,18 @@ public boolean passThroughForwardWall(int range) {
         }
 
         Location foundBlock = hitBlox.getCurrentBlock();
+        //actual max and min points of the world are 17 blocks away from the visible border
+        final int maxX = getWorld().getMaximumPoint().getX() - 17;
```

Comment:
Would it be alright to add a size setting for worldborder in the config file? I don't think worldedit touches the level.dat file where the worldborder size is and i can't find another way to access it.

Reply:
No, that isn't acceptable, since it can be changed in-game quite easily. That information should just be part of the Minecraft `World` type, but I'm unsure.